src/azul/plugins/repository/dss/__init__.py

```diff
@@ -93,11 +96,17 @@ def lookup_source_id(self, name: SimpleSourceName) -> str:
     def dss_client(self):
         return client(dss_endpoint=config.dss_endpoint)
 
+    @cached_property
```

Comment:
`config.dss_query_prefix` is cheap so there is no point in caching it. OTOH, doing so only makes mocking harder.

Last but not least, there is only one reference to this so even a normal property seems like overkill.